### ISSUE TITLE
about the flops in your paper.

### ISSUE THREAD

**Issue by suzhenghang** [mit-han-lab/temporal-shift-module#146]
hello，as marked in your paper， the flops and params of eco is 64g & 47.5M，could you share the code how to get the results?

**Comment by Nauman007** [comment]
Yes, need to get these results also... @suzhenghang  did u find something regarding FLOPS and params?
